examples/operator/101_initial_cluster.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     vtbackup: vitess/lite:latest
     vtorc: vitess/lite:latest
     mysqld:
-      mysql80Compatible: mysql:8.0.30
+      mysql80Compatible: mysql:8.0.40
     mysqldExporter: prom/mysqld-exporter:v0.11.0
```

Comment:
This has to match what we are using on the vtop repository. From @GrahamCampbell comment on https://github.com/planetscale/vitess-operator/pull/653
> mysqld_exporter to 0.14.0 (not to the latest 0.16.0 because of a breaking change in 0.15.0 which requires some effort to work around - for a different PR).

Reply:
Updated the exporter everywhere as well.